src/main/io/osd.c

```diff
@@ -645,9 +645,9 @@ static void osdDrawElements(void)
       return;
 
 #ifdef CMS
-    else if (sensors(SENSOR_ACC) || displayIsGrabbed(osdDisplayPort))
```

Comment:
Actually, looking again, the test for `displayIsGrabbed` is nonsensical as the call to `osdDrawElements` is guarded by this test.

(the OSD display being grabbed in this case is indicative of being in the CMS menu)